packages/daemon/src/service/pipeline.ts

```diff
@@ -230,26 +238,34 @@ export class PipelineService {
       // ensure the model dir exists
       await fs.ensureDir(modelPath);
 
+      emitJobEvent(PipelineStatus.RUNNING, 'dataCollect', 'start');
       // run dataCollect to download dataset.
       await runnable.start(plugins.dataCollect.plugin, getParams(plugins.dataCollect.params, {
```

Comment:
Could you please wrap this function with start/end emits?

Reply:
Done.